python/src/aiconfig/default_parsers/palm.py

```diff
@@ -138,7 +139,7 @@ async def run_inference(
             )
         )
 
-        # TODO: check api key here
+        set_api_key("PALM_KEY")
```

Comment:
Is this the standard name for the key from palm docs?

Reply:
It's GOOGLE_API_KEY for gemini and YOUR_API_KEY for palm (https://ai.google.dev/palm_docs/text_quickstart)

We should standardize this anyway instead of having the user rely on a random assortment of environment variables and hope

Reply:
We could standardize this to use the same as Gemini, especially since Gemini will be a core model parser